pingcap/tidb-operator
# Not suspend a component when the component is scaling or upgrading

### What problem does this PR solve?


If a component is scaling or upgrading, the statefulset may be upgrading, so we can' delete the sts at this time.

### What is changed and how does it work?


### Code changes


### Tests



Suspend a cluster when tikv is upgrading:
1. Deploy a cluster and upgrade tikv.
2. Suspend this cluster.
3. Check that all pods of this cluster are deleted. and tikv pods are deleted after the upgrade complete


### Side effects


### Related changes


### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```
